src/DssSpell.sol

```diff
@@ -39,11 +54,155 @@ contract DssSpellAction is DssAction {
+    address internal constant FRONTIERRESEARCH   = 0xA2d55b89654079987CF3985aEff5A7Bd44DA15A8;
+    address internal constant CHRISBLEC          = 0xa3f0AbB4Ba74512b5a736C5759446e9B50FDA170;
+    address internal constant CODEKNIGHT         = 0x46dFcBc2aFD5DD8789Ef0737fEdb03489D33c428;
+    address internal constant ONESTONE           = 0x4eFb12d515801eCfa3Be456B5F348D3CD68f9E8a;
+    address internal constant PVL                = 0x6ebB1A9031177208A4CA50164206BF2Fa5ff7416;
+    address internal constant CONSENSYS          = 0xE78658A8acfE982Fde841abb008e57e6545e38b3;
 
     function actions() public override {
 
+        // ----- Core Unit DAI Budget Streams -----
+
+        // VEST.restrict( Only recipient can request funds
+        //     VEST.create(
+        //         Recipient of vest,
+        //         Total token amount of vest over period,
+        //         Start timestamp of vest,
+        //         Duration of the vesting period (in seconds),
+        //         Length of cliff period (in seconds),
+        //         Manager address
+        //     )
+        // );
+
+        // DUX-001 | 2023-02-01 to 2024-01-31 | 1,611,420 DAI | 0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad
+        // https://vote.makerdao.com/polling/QmdhJVvN#vote-breakdown
+        MCD_VEST_DAI.restrict(
+            MCD_VEST_DAI.create(
+                DUX_WALLET,
+                1_611_420 * WAD,
+                FEB_01_2023,
+                JAN_31_2024 - FEB_01_2023,
+                0,
+                address(0)
+            )
+        );
+
+        // SES-001 | 2023-02-01 to 2024-01-31 | 3,199,200 DAI | 0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6
+        // https://vote.makerdao.com/polling/QmegsRNC#vote-breakdown
+        MCD_VEST_DAI.restrict(
+            MCD_VEST_DAI.create(
+                SES_WALLET,
+                3_199_200 * WAD,
+                FEB_01_2023,
+                JAN_31_2024 - FEB_01_2023,
+                0,
+                address(0)
+            )
+        );
+
+        // ------ Core Unit MKR Transfers -----
+        // DECO-001 - 125.0 MKR - 0xF482D1031E5b172D42B2DAA1b6e5Cbf6519596f7
+        // https://mips.makerdao.com/mips/details/MIP40c3SP36
+        MKR.transfer(DECO_WALLET, 125 * WAD);
+
+        // RISK-001 - 175.00 MKR - 0x5d67d5B1fC7EF4bfF31967bE2D2d7b9323c1521c
+        // https://mips.makerdao.com/mips/details/MIP40c3SP25
+        MKR.transfer(RISK_WALLET, 175 * WAD);
+
+        // ORA-001 - 843.69 MKR - 0x2d09B7b95f3F312ba6dDfB77bA6971786c5b50Cf
+        // https://forum.makerdao.com/t/psa-oracle-core-unit-mkr-distribution/19030
+        MKR.transfer(ORA_WALLET, 843.69 * 100 * WAD / 100);
+
+        // ----- MKR Vesting Stream ------
+        // Increase allowance by new vesting delta
+        MKR.approve(address(MCD_VEST_MKR), MKR.allowance(address(this), address(MCD_VEST_MKR)) + 675 ether);
+
+        MCD_VEST_MKR.restrict(
```

Comment:
I'm OK with unrestricted for this by the way.

I checked the chainlog address and it is DssVestTransferrable, I just wanted to confirm that this vest 'usr' can be transferred to a different address in the future if I wanted to.

Reply:
If security is better (somehow) for restricted we can do that though

Reply:
> I'm OK with unrestricted for this by the way. If security is better (somehow) for restricted we can do that though.

I believe this is just personal preference regarding what addresses can call `vest` for you.

> I checked the chainlog address and it is DssVestTransferrable, I just wanted to confirm that this vest 'usr' can be transferred to a different address in the future if I wanted to.

`Transferrable` refers to transfering the MKR instead of minting it.  This allows the `usr` to move the vest to a different address: https://github.com/makerdao/dss-vest/blob/master/src/DssVest.sol#L378-L383